### ISSUE TITLE
Docs: "Returns" section of imopen missing

### ISSUE THREAD

**Issue by sausix** [imageio/imageio#993]
There's absolutely no hint about the return type of v3.imopen in the documentation.

[imageio/core/imopen.py](https://github.com/imageio/imageio/blob/master/imageio/core/imopen.py#L71)

I already know it depends on the plugin class. It returns `imageio.plugins.pillow.PillowPlugin` in my case.

Thanks for the great project.

**Comment by FirefoxMetzger** [comment]
Good catch; we should probably add a sentence or two about this in the docs. Meanwhile, `imopen` returns an instance of the plugin class. Which one gets chosen depends on the image format, what plugins you have installed, and whether or not you overwrite auto-selection (using the `plugin=` kwarg).

To get a sense of which plugin is chosen when, check out our list of supported formats: https://imageio.readthedocs.io/en/stable/formats/index.html#all-formats
It lists the order in which plugins are tried for each supported format and the first installed plugin (that declares that it can handle the file) will be chosen.
